bugbug/model.py

```diff
@@ -498,6 +498,7 @@ def train(self, importance_cutoff=0.15, limit=None):
             y_pred_filter = []
             classified_indices = []
             for i in range(0, len(y_test)):
+                argmax = np.zeros(len(y_pred_probas), dtype=float)
```

Comment:
Why is this needed?

Reply:
ohh thats something i tried when i was getting that "IndexError: invalid index to scalar variable." i will revert it back!